google/cloud/forseti/common/util/metadata_server.py

```diff
@@ -20,6 +20,9 @@
 import httplib
 import socket
 
+from google.auth.compute_engine import _metadata
```

Comment:
I'm a little concerned about relying on a private module, as they could change the implementation at some point, I'd like to have some test to validate this is working as intended.